crates/pathfinder/src/sequencer.rs

```diff
@@ -185,9 +187,13 @@ impl Client {
             ))
             .send()
             .await?;
-        let resp = parse_raw(resp).await?;
-        let resp = resp.bytes().await?;
-        Ok(resp)
+        let def = parse::<reply::ContractDefinition>(resp).await?;
+        Ok(ContractDefinition {
+            abi: def.abi.to_string(),
```

Comment:
If it really is a String (which I think is a mistake, it should be serde_json::value::RawValue), it should just be moved here.

Reply:
I followed the path Mirko took:
- sequencer type contains `Box<RawValue>`, this one is only deserialized,
- core type contains `String`, this one is serialized (ultimately).

Reply:
Let's stick to String for the time being as per earlier comment.